resources/scripts/ttx_diff.py

```diff
@@ -413,19 +413,19 @@ def name_id_to_name(ttx, xpath, attr):
         if attr is None:
             if el.text is None:
                 continue
-            name_id = int(el.text)
+            name_id = el.text
             # names <= 255 have specific assigned slots, names > 255 not
-            if name_id <= 255:
+            if int(name_id) <= 255:
                 continue
-            el.text = id_to_name[el.text].strip()
+            el.text = id_to_name.get(name_id, name_id).strip()
         else:
             if attr not in el.attrib:
                 continue
             # names <= 255 have specific assigned slots, names > 255 not
-            name_id = int(el.attrib[attr])
-            if name_id <= 255:
+            name_id = el.attrib[attr]
+            if int(name_id) <= 255:
                 continue
-            el.attrib[attr] = id_to_name[el.attrib[attr]].strip()
+            el.attrib[attr] = id_to_name.get(name_id, name_id).strip()
```

Comment:
I suggest we might consider inserting a note that it's a bad name, e.g. let the default be something like `f"InvalidNameId {name_id}" or similar

Reply:
Done